tox.ini

```diff
@@ -7,7 +7,7 @@ deps=
     -rrequirements/compiled.txt
     -rrequirements/dev.txt
 commands =
-    coverage run manage.py test --noinput -v2 kuma
+    coverage run manage.py test --noinput -v2 {posargs:kuma}
     coverage report
 setenv =
     FORCE_DB = 1
```

Comment:
I think this `FORCE_DB` can go away now. That's a test-utils only thing iirc. The nose test runner has something equivalent but it's different... you have to tell it to reuse the db explicitly.